### ISSUE TITLE
The Guide does not make scenes 

### ISSUE THREAD

**Issue by GamingUnleashed** [Clownacy/Sonic-2-Clone-Driver-v2#12]
please fix it

**Comment by Clownacy** [comment]
You'll have to give some examples of it not making any sense: I just followed the Sonic 2 installation guide a couple of days ago and it seemed to be correct.

**Comment by GamingUnleashed** [comment]
Why does it say delete sound folder? & where’s the sonic 1 unused ram

**Comment by Clownacy** [comment]
You're right: the guide shouldn't have said to delete the sound folder. That was a leftover from an older version of the guide.

The unused block of RAM from Sonic 1 is the one labelled 'FFFFF100-$FFFFF5FF ; unused, leftover from the Sonic 1 sound driver (and used by it when you port it to Sonic 2)'. That RAM is also used by a bugfix for 2-player mode, so you will either have to disable that bugfix or find RAM elsewhere (such as by decompressing the collision arrays and loading them from ROM, allowing the RAM used by 'Primary_Collision' and 'Secondary_Collision' to be used by the sound driver instead).

**Comment by GamingUnleashed** [comment]
I will try it

**Comment by GamingUnleashed** [comment]
Can you make a guide of how you do it?

**Comment by GamingUnleashed** [comment]
![Image](https://github.com/user-attachments/assets/2b630145-f0d1-4746-90be-408b3109ea20) what do i do
